### PR TITLE
Fixed bug in ResetPasswordView. Fixed #1391.

### LINKED ISSUE #1391: ResetPasswordView does not work.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: tests/users/test_views.py [hunk 1]
```diff
@@ -57,6 +57,12 @@ def test_activate(self):
         response = self.client.get('/user/1/status/activate/')
         self.assertEqual(response.status_code, 302)
 
+    def test_reset_password(self):
+        self.admin.default_password = new_password = gen_password()
```
Comment (ostcar) [inline tests/users/test_views.py]: Please use a static password and not a generated password here. You don't test gen_password, but the reset_password-view. If the function gen_password is broken, this test could fail.

If you want to test the gen_password function, then please write its own test.

**Comment by ostcar** [comment]
Please fix travis, and then you can merge.

I can live with it if you don't change the test.

File: tests/users/test_views.py (hunk 1 above)
Comment (normanjaeckel) [inline tests/users/test_views.py]: Done.

**Comment by ostcar** [comment]
ok, merge after travis is fixed.

**Comment by ostcar** [comment]
Optional: rebase

**Pull request merged**